gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -1994,6 +1994,74 @@ static PyObject* GemRB_Control_SetVarAssoc(PyObject* self, PyObject* args)
 	Py_RETURN_FALSE;
 }
 
+PyDoc_STRVAR( GemRB_Window_GetVar__doc,
+"===== Window_GetVar =====\n\
+\n\
+**Prototype:** GWindow.GetVar (var)\n\
+\n\
+**Metaclass Prototype:** GetVar (var)\n\
+\n\
+**Description:** Get the value of a window variable.\n\
+Variables are set as a result of calling SetVarAssoc on a GControl.\n\
```

Comment:
I suggest you either mention it here or add both Window_SetVar and SetVarAssoc to a "See also" section. Like before core may change the values as well.